core/templates/domain/learner_dashboard/learner-dashboard-backend-api.service.ts

```diff
@@ -350,7 +350,7 @@ export class LearnerDashboardBackendApiService {
     return new Promise((resolve, reject) => {
       this.http.get<SubtopicMasteryDict>(
         AppConstants.SUBTOPIC_MASTERY_DATA_URL_TEMPLATE, {
-          params: { comma_separated_topic_ids: topicIds }}).toPromise()
+          params: { selected_topic_ids: JSON.stringify(topicIds) }}).toPromise()
```

Comment:
Can we refactor this in a way that instead of passing the comma-separated string into this function we pass the list of strings (list of topic IDs)?

Reply:
Done.

Reply:
Done.